draft-ietf-tls-exported-authenticator.md

```diff
@@ -256,7 +256,7 @@ requirements of a Certificate message in the negotiated version of TLS.  In
 particular, the certificate chain MUST be valid for the a signature algorithms
 indicated by the peer in the "signature_algorithms" and "signature_algorithms_cert"
 extension, as described in Section 4.2.3 of {{!TLS13}} for TLS 1.3 or the "signature_algorithms" extension
-from Sections 7.4.2 and 7.4.6 of {{!RFC5246}} for TLS 1.2.
+from Sections 7.4.2 and 7.4.6 of {{?RFC5246}} for TLS 1.2.
```

Comment:
Can you have a MUST-level requirement described in an informative reference?

Reply:
Good question. The complaint idnits gave was that RFC 5246 was obsoleted by RFC 8446.

Reply:
Yeah, but this extension still applies for TLS 1.2, so referencing 1.2 is mostly unavoidable.  Nits don't mean that you have to fix them, just that you need a good reason for diverging.  You've got one.